locale/fi/LC_MESSAGES/django.po

```diff
@@ -1021,6 +1303,13 @@ msgstr ""
 msgid "View the participants"
 msgstr "Katso osallistujat"
 
+#~ msgid "Trying to save event with deprecated keywords "
+#~ msgstr "Yritetään tallentaa tapahtumaa vanhentuneilla avainsanoilla"
+
+#, python-brace-format
+#~ msgid "Registration with id {pk} doesn't exist."
+#~ msgstr "Ilmoittautumista tunnuksella {pk} ei ole olemassa."
+
 #~ msgid "Admins"
 #~ msgstr "Ylläpitäjät"
```

Comment:
These could be cleaned away I think (while you're checking those upper translations)